Details Fragment Refactoring

BadLog1n opened the pull request.

BadLog1n committed:
Details Fragment Refactoring

BadLog1n committed:
Return description image

MeeWeeL commented on app/src/main/res/layout/new_details_fragment.xml:
```diff
@@ -1,175 +1,106 @@
     tools:context=".presentation.detailsFragment.DetailsFragment">
 
-    <androidx.core.widget.NestedScrollView
+    <ScrollView
```
Почему перевёл на обычный ScrollView?  NestedScrollView предусматривает наложенные жесты, то есть избавляет от конфликтов вложенной прокрутки

MeeWeeL commented on app/src/main/res/values/styles.xml:
```diff
@@ -457,6 +457,47 @@
     <!--  Age Rate  -->
     <!--  Genre  -->
     <!--  Description Background Image  -->
+
```
Страшно

MeeWeeL commented on app/src/main/java/com/meeweel/anilist/presentation/detailsFragment/DetailsFragment.kt:
```diff
@@ -50,33 +52,41 @@ class DetailsFragment : Fragment(R.layout.new_details_fragment) {
 
     @SuppressLint("SetTextI18n")
     private fun populateData(animeData: Anime) {
+        val isRussian = App.ContextHolder.context.resources.getBoolean(R.bool.isRussian)
```
У фрагмента есть контекст

BadLog1n committed:
Details refactoring

1. Styles cosmetic changes
2. add appcompat in layout
3. remove  App.ContextHolder
